gitops/eshop/tsb-config.yaml

```diff
@@ -116,24 +116,24 @@ items:
           - route:
               host: gitops-eshop/products.gitops-eshop.svc.cluster.local
               port: 80
-        # rateLimiting:
-        #   settings:
-        #     rules:
-        #       - dimensions:
-        #           - remoteAddress:
-        #               value: '*'
-        #         limit:
-        #           requestsPerUnit: 3
-        #           unit: SECOND
-      # waf:
-      #   rules:
-      #     - Include @recommended-conf         # Recommended basic setup
-      #     - SecRuleEngine On                  # Override DetectionOnly mode in @recommended-conf
-      #     - SecResponseBodyAccess Off         # Override to not process response bodies
-      #     - Include @crs-setup-conf           # Initialize the Core Rule Set
-      #     - Include @owasp_crs/*.conf         # Load the Core Rule Set
-      #     # Custom rule to deny requests with query parameter 'id=0'
-      #     - SecRule ARGS:id "@eq 0" "id:1, phase:1,deny, status:403,msg:'Invalid id',log,auditlog"
+        rateLimiting:
+          settings:
+            rules:
+              - dimensions:
+                  - remoteAddress:
+                      value: '*'
+                limit:
+                  requestsPerUnit: 3
+                  unit: SECOND
+      waf:
+        rules:
+          - Include @recommended-conf         # Recommended basic setup
+          - SecRuleEngine On                  # Override DetectionOnly mode in @recommended-conf
+          - SecResponseBodyAccess Off         # Override to not process response bodies
+          - Include @crs-setup-conf           # Initialize the Core Rule Set
+          - Include @owasp_crs/*.conf         # Load the Core Rule Set
+          # Custom rule to deny requests with query parameter 'id=0'
+          - SecRule ARGS:id "@eq 0" "id:1, phase:1,deny, status:403,msg:'Invalid id',log,auditlog"
```

Comment:
can we also add 
```
  - SecRule REQUEST_HEADERS:user-agent "@contains curl" "id:2, pass, log,
    logdata:'someone used curl to access'"
```
also something i want to add it is a `json` payload sanitizer using `waf`...  i.e. just to outline the functionality and ability to tamper the body.

Reply:
There is a traffic generator installed for the eshop and this will introduce a lot of noise, because the traffic generator uses curl.. Processing the user-agent is already covered in an example in the eshop README, so I'd rather not add this rule.

Reply:
yeah.. do forget about the noise, agree, lets comment out the rule.